docs/spl-docs/evaluation-functions/time-functions.md

```diff
@@ -0,0 +1,71 @@
+# Date and Time functions
+
+_This list includes functions designed for handling date and time operations._
+
+
+## **now()**
+
+This function returns the current date and time.
+
+### Usage
+
+You can use this function to get the current timestamp in your queries. It is commonly used with the `eval` command to add the current time to your data.
+
+### Example
+
+1. Using the `now` function with the `eval` command to add the current timestamp to each event:
+```spl
+... | eval current_time=now()
+```
+
+2. When working with time data in logs or events, it's common to need to calculate the time elapsed between when an event occurred and the current time. This can be particularly useful for monitoring systems, where understanding the recency of events is crucial. 
+
+Imagine you're monitoring a system that logs events with high-precision timestamps, and you're interested in categorizing events based on how recently they occurred. Here's how you could achieve this using the `now` command:
+
+```spl
+... | eval current_time_ms=now()*1000, time_difference_ms=current_time_ms - timestamp
+| eval time_category=if(time_difference_ms <= 300000, "within 5 minutes",
+	if(time_difference_ms <= 600000, "within 10 minutes",
+	if(time_difference_ms <= 1800000, "within 30 minutes", "older than 30 minutes")))
+```
+
+This SPL command calculates the time difference in milliseconds between the current time and the event timestamp, then uses nested `if` statements to categorize each event based on this time difference.
+
+## **strftime(&lt;epoch_time&gt;, &lt;format&gt;)**
+
+This function formats an epoch time value as a string according to the specified format.
+
+### Usage
+
+The `<epoch_time>` argument is the time in epoch format that you want to format. The `<format>` argument is the format string that defines how the output should be formatted.
+
```

Comment:
I have specified the invalid inputs case and added use-case examples, for specifying the valid specifiers `format` can contain, there are multiple specifiers in the `formatTime` method (linked below) - how should we list these? Should we have a table with all the valid specifiers? Should the table contain examples, or written description for each or both? 
https://github.com/siglens/siglens/blob/726bd106f7a40a9e2c9d44def394faecd9837a92/pkg/segment/structs/evaluationstructs.go#L1624-L1681

Reply:
If there's already stable, correct documentation for these, we can link to that; otherwise, we should write our own. A table sounds like a good idea to me. We want to communicate to the reader accurately and concisely. I would guess that would mean a written description and possible examples for some of them, but use your judgement